runner.yml.dist

```
@@ -24,6 +24,7 @@ drupal:
     - "./vendor/bin/drush en oe_webtools oe_webtools_laco_service -y"
     - "./vendor/bin/drush en oe_webtools oe_webtools_laco_widget -y"
     - "./vendor/bin/drush en oe_webtools oe_webtools_cookie_consent -y"
+    - "./vendor/bin/drush en oe_webtools oe_webtools_social_share -y"
```

```suggestion
    - "./vendor/bin/drush en oe_webtools_social_share -y"
```
It should be fixed in all lines except the first 😄

---

Indeed. Let's put all our modules in one line